README.md

```diff
@@ -1,50 +1,78 @@
 
-The Shop at Home theme is used throughout the app. View it live at <https://www.shopathome.dev>
+- The Frontend is built using 4 JavaScript frameworks (Angular, React, Svelte, Vue) and is deployed to **Azure Static Web Apps**
+- The Backend server running on Fastify is deployed to **Azure Container Apps**
 
-[![Deploy to Azure button](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/?feature.customportal=false&WT.mc_id=academic-0000-jopapa#create/Microsoft.StaticApp)
+## Run Shop at Home locally
 
-## Learn how
-
-Learn how to [Publish an Angular, React, Svelte, or Vue JavaScript app and API with Azure Static Web Apps](https://docs.microsoft.com/learn/modules/publish-app-service-static-web-app-api/?WT.mc_id=academic-0000-jopapa)
-
-## Contents
+### Prerequisites
+- A GitHub account
+- [Node.js and Git](https://nodejs.org/)
+- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=academic-0000-jopapa) installed
+- The [SWA CLI](https://www.npmjs.com/package/@azure/static-web-apps-cli) installed
+- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=academic-0000-jopapa) installed
 
-The apps written in the following JavaScript frameworks/libraries:
+See instructions to install, setup and run the application locally in the **README.md** file located at the root of the respective folders.
 
 | Folder          | Description                                                                          | Live Demo                                                |
 | --------------- | ------------------------------------------------------------------------------------ | -------------------------------------------------------- |
 | **angular-app** | [Sample Angular app](https://github.com/johnpapa/shopathome/blob/master/angular-app) | [angular.shopathome.dev](https://angular.shopathome.dev) |
-| **api**         | [Sample Azure Functions app](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
+| **api**         | [Fastify API](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
 | **react-app**   | [Sample React app](https://github.com/johnpapa/shopathome/blob/master/react-app)     | [react.shopathome.dev](https://react.shopathome.dev)     |
 | **svelte-app**  | [Sample Svelte app](https://github.com/johnpapa/shopathome/blob/master/svelte-app)   | [svelte.shopathome.dev](https://svelte.shopathome.dev)   |
 | **vue-app**     | [Sample Vue app](https://github.com/johnpapa/shopathome/blob/master/vue-app)         | [vue.shopathome.dev](https://vue.shopathome.dev)         |
 
-## Prerequisites
+## Run Shop at Home on Azure 
+> [!NOTE]
+> Ensure you fork the project to your GitHub account before you start deploying.

```

Comment:
You won't need to fork this repo while using App Spaces. We fork it for you.

Reply:
Right - I've removed the note! Thank you